src/graphs/BasicGraphs.jl

```diff
@@ -119,17 +133,24 @@ add_vertex!(g::HasVertices; kw...) = add_part!(g, :V; kw...)
 """
 add_vertices!(g::HasVertices, n::Int; kw...) = add_parts!(g, :V, n; kw...)
 
+""" Add vertices with preallocated src/tgt indexes
+"""
+function add_vertices_with_indices!(g::HasVertices, n::Int, k::Int; kw...)
```

Comment:
Are there unit tests for this exported function? Warning from CodeCov below suggests now.